content/es/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Instalar y Configurar kubectl en macOS
+content_type: task
+weight: 10
+---
+
+## {{% heading "prerequisites" %}}
+
+Se debe utilizar la versión de kubectl con la menor diferencia de versión de respecto de 
+su clúster. Por ejemplo, un cliente con versión v{{< skew currentVersion >}} se puede comunicar
+con los siguientes versiones de plano de control v{{< skew currentVersionAddMinor -1 >}}, 
+v{{< skew currentVersionAddMinor 0 >}}, and v{{< skew currentVersionAddMinor 1 >}}.
+Utilizar la última versión compatible de kubectl evita posibles errores.
+
+## Instalar kubectl en macOS
+
+Existen los siguientes métodos para instalar kubectl en macOS:
+
+- [Instalar kubectl en macOS](#instalar-kubectl-en-macos)
+  - [Instalación del binario para macOS con Curl](#instalación-del-binario-para-macos-de-kubectl-con-curl)
+  - [Instalar con Homebrew en macOS](#instalar-utilizando-homebrew-en-macos)
+  - [Instalar con Macports en macOS](#instalar-con-macports-en-macos)
+- [Verificar la configuración de kubectl](#verificar-la-configuración-de-kubectl)
+- [Configuraciones y plugins opcionales para kubectl](#configuraciones-opcionales-y-plugins-de-kubectl)
+  - [Habilitar el autocompletado de la shell](#instalar-bash-complete)
```

Comment:
We should use the same words as the subtitle below to have the reference working:

```suggestion
  - [Habilitar el autocompletado de la shell](#habilitar-el-autocompletado-en-la-shell)
```

Reply:
> We should use the same words as the subtitle below to have the reference working:

I will check it but there is a little difference because for this example we need install bash-complete to enable the feature. Thats why I use it.

Let me see and I will share with you the related information.

Reply:
Yes, I agree that `bash-complete` is needed to enable that feature.

The suggestion here is to fix the "hashtag link". We need to use the `#same-words-as-subtitle` to create a reference to this subtitle below. You can build the Kubernetes website locally with Hugo to test if the "hashtag link" works.

You can review more about Hugo links here:
https://gohugo.io/content-management/cross-references/#heading-ids

Or this other PR where another team had the same issue:
https://github.com/kubernetes/website/pull/37437/files